docs/usage.md

```diff
@@ -178,7 +178,7 @@ By default, if you don't provide features, for non-array data the workflow will
 
 The pipeline produces an R markdown file which, if you're proficient in R, you can use to tweak the report after it's generated (**note**- if you need the same customisations repeatedly we would recommend you supply your own template using the `report_file` parameter).
 
-To work with R markdown files you will need Rstudio. You will also need to have the ShinyNGS R module [installed](https://github.com/pinin4fjords/shinyngs#installation), since it supplies a lot of the accessory plotting functions etc that you will need. The exact way you will do this may depend on your exact systems, but for example
+To work with R markdown files you will need RStudio. You will also need to have the ShinyNGS R module [installed](https://github.com/pinin4fjords/shinyngs#installation), since it supplies a lot of the accessory plotting functions etc. that you will need. The exact way to set this up may depend on your specific system, but is likely similar to the following example:
```

Comment:
Sorry, I'm happy with the original wording.

Reply:
Please don't resolve without addressing the comments

Reply:
> Please don't resolve without addressing the comments

Hi. Please check. I just kept the typoes (e.g. changing "analyis" to "analysis"), and I changed all my previous edits, due to your comment. If I missed something, please let me know.